src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/graphs/BaseGraph.java

```diff
@@ -376,7 +376,7 @@ public final void printGraph(final PrintStream graphWriter, final boolean writeH
         for( final E edge : edgeSet() ) {
             final String edgeString =  String.format("\t%s -> %s ", getEdgeSource(edge).toString(), getEdgeTarget(edge).toString());
             final String edgeLabelString;
-            if (edge.getMultiplicity() > 0 && edge.getMultiplicity() <= pruneFactor){
+            if (edge.getMultiplicity() > 0 && edge.getMultiplicity() < pruneFactor){
```

Comment:
Yes, everything should change to match `LowWeightChainPruner` because it is used in `HaplotypeCaller`'s assembly step.